rules/Php73/Rector/FuncCall/ArrayKeyFirstLastRector.php

```diff
@@ -130,7 +130,7 @@ public function refactor(Node $node): ?Node
 
         $this->removeNode($node);
 
-        return $node;
+        return null;
```

Comment:
this part need to be rolled back as it change next func call before remove node, so it change next node data that need to be reconnected.

Reply:
Reverted with add comment note for it https://github.com/rectorphp/rector-src/pull/3558/commits/a5944d3475a671c1c7b1009ba320d80493e32c8f